fapi-client/src/main/scala/com/gu/facia/api/utils/Backfill.scala

```diff
@@ -40,10 +39,11 @@ object BackfillResolver {
         } yield {
           backfillContent.map(CuratedContent.fromTrailAndContent(_, TrailMetaData.empty, None, collectionConfig))
         }
-      case CollectionBackfill(parentCollectionId) =>
+      case CollectionBackfill(parentCollectionId) => {
         for {
           collection <- FAPI.getCollection(parentCollectionId)
           curatedCollection <- FAPI.liveCollectionContentWithSnaps(collection, adjustSearchQuery, adjustItemQuery)
         } yield curatedCollection
+      } recover { err => List() }
```

Comment:
This feels a bit hidden and easy to miss! I think you should put the future into it's own `val` and then do the recover in that. And in there add logging, otherwise we will never know how often this happens.